### PR TITLE
[docs] test config-reference formatting for sidebar

### PR DESCRIPTION
**Pull request by sarah11918**
## Changes

Changes `Prefetch` and `i18n` into `heading`s to see if I can get the docs sidebar nested properly

## Testing

This is a test! The automated PR for docs will let me test the output

## Docs

Just docs!

### CODE REVIEW

**Review by ematipico — APPROVED** [review]

**Review by delucis — CHANGES_REQUESTED** [review]
Don’t think this will do what you want!

**Comment by delucis** [comment]
(Speed reviewed in case Ema’s approval made you merge this 😅)

I’m pretty sure our logic in docs only distinguishes between `@heading` and blocks without `@heading`. So these you’ve added would pull these out to be h2s like this:

image image

Breaking for lunch, but we can take a look at a solution when I’m back. (FWIW the current version is potentially OK I think as it matches e.g. how we handle the `build.*` headings.)

**Comment by sarah11918** [comment]
For Prefetch I think that works, if it's moved to other things that work that way! I'd really just want the experimental flag to show up as a one-liner, I think though:

image
image

File: packages/astro/src/@types/astro.ts [hunk 1]
```diff
@@ -538,6 +538,7 @@ export interface AstroUserConfig {
 
 	/**
 	 * @docs
+	 * @kind heading
 	 * @name prefetch
```
Comment (sarah11918) [inline packages/astro/src/@types/astro.ts:542]:
```suggestion
	 * @name Prefetch Options
```

**Comment by delucis** [comment]
Ah, yes, that makes sense. In that case, we probably also need to move the whole `prefetch` block AFTER the build options?

**Comment by sarah11918** [comment]
Yes, but I'm still not sure how to make the i18n options not show up in the sidebar

**Review by ElianCodes — APPROVED** [review]
LGTM!

File: packages/astro/src/@types/astro.ts [hunk 2]
```diff
@@ -1481,6 +1484,7 @@ export interface AstroUserConfig {
 
 			/**
 			 * @docs
+		   * @kind h4
 			 * @name experimental.i18n.fallback
 			 * @type {Record<string, string>}
 			 * @version 3.5.0
```
Comment (delucis) [inline packages/astro/src/@types/astro.ts:1490]: Now that this is an `<h4>`, the `#### Example` in the `@description` should probably get an extra `#`.

**Review by delucis — APPROVED** [review]
Looks great @sarah11918! Glad we’re no longer JSBlocked 🚀

**Comment by sarah11918** [comment]
Thanks for your help with this, @delucis ! This is now `headings` in the right direction!

**Pull request merged**